server/server.go

```diff
@@ -97,6 +97,8 @@ type Info struct {
 	LeafNodeURLs []string `json:"leafnode_urls,omitempty"` // LeafNode URLs that the server can reconnect to.
 }
 
+type gossipURLs map[string]int
```

Comment:
maybe move to util where it's functions are?

Reply:
I wonder even if I make that a type. It does not prevent one from using map[string]int directly anyway. But if I keep as a type, should I name it differently? Should it be more mapOfURLsWithRefcount or something like that? (I am very bad at naming)

Reply:
I'm fine with the current name. Maybe refCountedUrlSet?

Reply:
I have renamed, moved to util and made the add/remove/get receivers for this new type. I think it is cleaner. I you could have another look, I would appreciate it.